app/models/workshop.rb

```diff
@@ -0,0 +1,5 @@
+class Workshop < ApplicationRecord
+  validates :title, presence:true
+  validates :description, presence:true
```

Comment:
you could write something like `validates :title, :description, presence: true` if you'd like

But, in any case, add spaces between `presence: true`.